.openpublishing.redirection.json

```diff
@@ -59,6 +59,10 @@
         {
             "source_path_from_root": "/docs/openai/azureai-openai-component.md",
             "redirect_url": "/dotnet/aspire/azureai/azureai-openai-component"
-        }
+        },
+        {
+            "source_path_from_root": "/docs/troubleshooting/allowunsecuretransport.md",
+            "redirect_url": "/dotnet/aspire//troubleshooting/allow-unsecure-transport."
```

Comment:
@IEvangelist this doesn't look right (trailing . and two //)

Reply:
Never mind ... I unblocked this. I found a appropriate mail enabled security group to use for links. I don't know why the aka.ms UI wouldn't let me change a link it let me create last week. That UX is so clunky :)